celavi/pylca_celavi/insitu_emission.py

```diff
@@ -198,11 +198,13 @@ def runner(tech_matrix,F,yr,i,j,k,final_demand_scaler,process,df_with_all_other_
     res = pd.DataFrame()
     res= solver_optimization(tech_matrix, F,process,df_with_all_other_flows)
     res['value'] = res['value']*final_demand_scaler
-    if res.empty == False:
+    if not res.empty:
        res.loc[:,'year'] =  yr
        res.loc[:,'facility_id'] = i
        res.loc[:,'stage'] = j
        res.loc[:,'material'] = k
+    else:        
+       print(f"Insitu emission optimization failed for {k} at {j} in {yr}")
```

Comment:
I updated the other optimization failed statements as well, for easier debugging

Reply:
I updated many print statements in this fashion for easier debugging.